docs/UserGuide.md

```diff
@@ -164,14 +164,13 @@ Format: `unflag NAME`
 Sorts clients such that flagged clients are displayed before unflagged clients.
 It then sorts all clients in HustleBook based on the parameter provided.
 
-Format: `sort PARAMETER`
+Format: `sort [meeting|name|prev|salary]`
 
-
-* `PARAMETER` you can use are: `meeting`,`name`, `prev` and `salary`
-* `PARAMETER` is not case-sensitive. 
-
-Example:
-* `sort salary`
+* `sort meeting` sorts clients by the date and time of the meeting. The next upcoming meeting will be displayed first.
```

Comment:
The second sentence seems a bit confusing. It would be better if you can rephrase it. Maybe one suggestion would be: `The earliest upcoming meeting will be displayed first.`

Reply:
That's is a better way to phrase it. Thanks! Was struggling a little on how I should phrase it.